README.md

```diff
@@ -62,6 +63,9 @@ Or
 ## Installation
 * `git clone <repository-url>` this repository
 * `cd ember-plaid-v2`
+* install bower through brew
+* `bower install ember`
```

Comment:
why are you installing bower? I thought we had completely removed bower dependencies. we should be using yarn instead.

Reply:
me and ryan ran into problems getting any of the ember cli commands to work (ex:ember serve). we only got things working after running those setup commands

Reply:
When we cloned the repo and tried to run the project, it was complaining that we were missing files in bower/ember.